src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -202,50 +201,54 @@ def __init__(self, target: Text,
 
         self._channel = cygrpc.AioChannel(_common.encode(target))
 
-    def check_connectivity_state(self, try_to_connect: bool = False
-                                ) -> grpc.ChannelConnectivity:
+    def get_state(self,
+                  try_to_connect: bool = False) -> grpc.ChannelConnectivity:
         """Check the connectivity state of a channel.
 
         This is an EXPERIMENTAL API.
 
+        It's the nature of connectivity states to change. The returned
+        connectivity state might become obsolete soon. Combining
```

Comment:
s/obsolete/stale/

Reply:
How about "if the channel reaches a stable connectivity state, it is guaranteed that the return value of this function will eventually converge to that state."

Reply:
Updated the docstring to your version 👍